tests/test_sqs/test_sqs.py

```diff
@@ -837,9 +837,10 @@ def test_delete_message_after_visibility_timeout():
     assert new_queue.count() == 0
 
 
+@skip_in_server_mode
 @mock_sqs
 def test_batch_change_message_visibility():
-    if os.environ.get('TEST_SERVER_MODE', 'false').lower() == 'true':
+    if settings.TEST_SERVER_MODE:
```

Comment:
If the function has been decorated with skip_test then you dont need the if statement right?